src/ctia/stores/es/indicator.clj

```diff
@@ -19,12 +12,15 @@
 
 (def ^{:private true} mapping "indicator")
 
+(defn long-id->short-id [long-id]
+  (id/short-id (id/long-id->id long-id)))
```

Comment:
Use _ctia.domain.id/str->short-id_ instead, or if you want this as an optimization, add it to ctia.domain.id.